app/controllers/users_controller.rb

```diff
@@ -10,6 +10,7 @@ def index
     lists_of_users
   end
 
+  # rubocop:disable Style/GuardClause
```

Comment:
Please fix the code instead of simply deactivating the linter. For more info on how to fix this issue, you may take a look at:

https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/GuardClause